### PR TITLE
HW1 -- yy3131

### PR DESCRIPTION
**Pull request by daisyye0730**

### CODE REVIEW

File: projects/Numpy2.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Numpy
```
Comment (timkpaine) [inline projects/Numpy2.md:1]: rename file to `numpy.md`

File: projects/numpy.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Numpy
```
Comment (timkpaine) [inline projects/numpy.md:1]: you don't need to use my `numpy.md`, use yours as `numpy.md`.

File: projects/Numpy2.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (timkpaine) [inline projects/Numpy2.md:2]: move to `projects/python/numpy.md`

**Review by timkpaine — CHANGES_REQUESTED** [review]
restore `tkp2108.md`

File: projects/Numpy2.md (hunk 1 above)
Comment (daisyye0730) [reply projects/Numpy2.md]: I can't name two projects both as numpy.py, so i am naming it as numpy_main_project

File: tkp2108.md [hunk 4]
```diff
@@ -4,7 +4,9 @@ Name: Tim Paine
 
 UNI: tkp2108
 
-Github: [link](https://github.com/timkpaine)
+GitHub: [link](https://github.com/timkpaine)
```
Comment (timkpaine) [inline tkp2108.md:7]: remove these changes

File: yy3131.md [hunk 5]
```diff
@@ -6,8 +6,10 @@ UNI: yy3131
 
 Github: [link](https://github.com/daisyye0730)
 
+GitHub README: [link](https://github.com/daisyye0730/daisyye0730/blob/main/README.md)
+
 Project Proposal
-1. Numpy flipping an array diagonally 
+[project](https://github.com/daisyye0730/project-proposals-s2023/blob/main/projects/python/numpy_main_project.md)
```
Comment (timkpaine) [inline yy3131.md:12]: use relative path like `./projects/python/numpy.md`

File: yy3131.md [hunk 6]
```diff
@@ -6,8 +6,10 @@ UNI: yy3131
...
 
 Project Contributions
-1. Numpy: sort an array of tuples given a certain index of tuple 
+[bug](https://github.com/daisyye0730/project-proposals-s2023/blob/main/projects/python/numpy.md)
```
Comment (timkpaine) [inline yy3131.md:15]: use relative path

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Pull request closed without merging**